deploy/Chart/templates/dashboard/rbac.yaml

```diff
@@ -10,7 +10,7 @@ metadata:
 rules:
   - apiGroups: ['api.ucp.dev']
     resources: ['*']
-    verbs: ['get', 'list']
+    verbs: ['*']
```

Comment:
Instead of `*` should we only have the ones **dashboard** needs?

Reply:
we also need `post` here (that was the bug). I don't have a strong preference between being explicit and using the wildcard. This is just our API and the tradeoffs are small.